### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by shuyama1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+* sql: made `settings.sql_server_audit_config.bucket` field in `google_sql_database_instance` to be optional. ([#13252](https://github.com/hashicorp/terraform-provider-google/pull/13252))
+* storagetransfer: supported in-place update for `schedule` in `google_storage_transfer_job` ([#13262](https://github.com/hashicorp/terraform-provider-google/pull/13262))
+
+BUG FIXES:
+* bigquery: fixed a permadiff on `labels` of `google_bigquery_dataset` when it is referenced in `google_dataplex_asset` ([#13333](https://github.com/hashicorp/terraform-provider-google/pull/13333))
+* compute: fixed an issue where `enable_dynamic_port_allocation` was not able to set to `false` in `google_compute_router_nat` ([#13243](https://github.com/hashicorp/terraform-provider-google/pull/13243))
+* cloudidentity: fixed an issue where where plan would fail when `cloud_identity_group_membership` had been removed outside terraform ([#13331](https://github.com/hashicorp/terraform-provider-google/pull/13331))
+* identityplatform: fixed issues with `identity_platform_config` creation ([#13301](https://github.com/hashicorp/terraform-provider-google/pull/13301))
```
Comment (slevenick) [inline CHANGELOG.md:45]: `google_identity_platform_config`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+* sql: made `settings.sql_server_audit_config.bucket` field in `google_sql_database_instance` to be optional. ([#13252](https://github.com/hashicorp/terraform-provider-google/pull/13252))
+* storagetransfer: supported in-place update for `schedule` in `google_storage_transfer_job` ([#13262](https://github.com/hashicorp/terraform-provider-google/pull/13262))
+
+BUG FIXES:
+* bigquery: fixed a permadiff on `labels` of `google_bigquery_dataset` when it is referenced in `google_dataplex_asset` ([#13333](https://github.com/hashicorp/terraform-provider-google/pull/13333))
+* compute: fixed an issue where `enable_dynamic_port_allocation` was not able to set to `false` in `google_compute_router_nat` ([#13243](https://github.com/hashicorp/terraform-provider-google/pull/13243))
+* cloudidentity: fixed an issue where where plan would fail when `cloud_identity_group_membership` had been removed outside terraform ([#13331](https://github.com/hashicorp/terraform-provider-google/pull/13331))
```
Comment (slevenick) [inline CHANGELOG.md:44]: What resource?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,51 @@
...
+* container: added `gateway_api_config` block to `google_container_cluster` resource for supporting the gke gateway api controller ([#13233](https://github.com/hashicorp/terraform-provider-google/pull/13233))
+* container: supported in-place update for `labels` in `google_container_node_pool` ([#13284](https://github.com/hashicorp/terraform-provider-google/pull/13284))
+* dataproc: added support for `SPOT` option for `preemptibility` in `google_dataproc_cluster` ([#13335](https://github.com/hashicorp/terraform-provider-google/pull/13335))
+* dlp: added field `deidentify_config.record_transformations.field_transformations` to `google_data_loss_prevention_deidentify_template` ([#13282](https://github.com/hashicorp/terraform-provider-google/pull/13282))
+* dlp: added field `deidentify_config.record_transformations.record_suppressions` to `google_data_loss_prevention_deidentify_template` ([#13300](https://github.com/hashicorp/terraform-provider-google/pull/13300))
+* dlp: added `version` field to `data_loss_prevention_inspect_template` resource ([#13366](https://github.com/hashicorp/terraform-provider-google/pull/13366))
```
Comment (slevenick) [inline CHANGELOG.md:35]: What resource?

**Review by slevenick — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (shuyama1) [reply CHANGELOG.md]: Fixed! Thanks!

**Pull request merged**